Makefile

```diff
@@ -576,7 +576,7 @@ lib/libmxnet.a: $(ALLX_DEP)
 	@mkdir -p $(@D)
 	ar crv $@ $(filter %.o, $?)
 
-lib/libmxnet.so: $(ALLX_DEP)
+lib/libmxnet.so: $(ALLX_DEP) sample_lib
```

Comment:
If it is not actually a dependency of libmxnet.so, we shouldn't put it here. Can we move it to .PHONY and/or all target instead?

Reply:
Addressed. Moved to all target.